sympy/combinatorics/perm_groups.py

```diff
@@ -694,6 +694,43 @@ def basic_transversals(self):
             self.schreier_sims()
         return self._transversals
 
+    def composition_series(self):
+        """
+        Add docs
+
+        """
+        if not self.is_solvable:
+            raise NotImplementedError('Group should be solvable')
+        der = self.derived_series()
+        series = [self]
+
+        for i in range(len(der)-1):
+            comp = []
+            low = der[i+1]
+            gns = low.generators
+            for elm in der[i].generators:
+                if low == der[i]:
+                    break
+                if not elm in low:
+                    elm = [elm]
+                    for pelm in elm:
+                        order = pelm.order()
+                        div = divisors(order)[1]
+                        l = Integer(log(order, div))
```

Comment:
This could be `multiplicity(div, order)`. Or `factorint(order)` would give both the prime factors and their multiplicity.